frontend/js/main.js

```diff
@@ -0,0 +1,2203 @@
+      $('#chckAll_' + tableNumber).prop('checked', false);
+    }
+  });
+}
+
+/**
+ * Function to edit column name for spanner table
+ *
+ * @param {html Element} editColumn
+ * @param {number} tableNumber
+ * @param {number} tableColumnNumber
+ * @return {null}
+ */
+const editSpannerColumnName = (editColumn, tableNumber, tableColumnNumber) => {
+  spannerCellsList[0] = editColumn;
+  columnNameVal = document.getElementById('columnNameText' + tableNumber + tableColumnNumber + tableColumnNumber).innerHTML;
+  initialColNameArray[tableNumber].push(columnNameVal);
+  currSeqId = '';
+  for (var x = 0; x < pkArray[tableNumber].length; x++) {
+    if (pkArray[tableNumber][x].Col == columnNameVal.trim()) {
+      currSeqId = pkArray[tableNumber][x].seqId;
+    }
+  }
+  if (notPrimary[tableNumber][tableColumnNumber] == true) {
+    spannerCellsList[0].innerHTML = `<span class="column left keyNotActive keyMargin keyClick" id='keyIcon${tableNumber}${tableColumnNumber}${tableColumnNumber}'>
+                                      <img src='./Icons/Icons/ic_vpn_key_24px-inactive.svg'>
+                                    </span>
+                                    <span class="column right form-group">
+                                      <input id='columnNameText${tableNumber}${tableColumnNumber}${tableColumnNumber}' type="text" value=${columnNameVal} class="form-control spanner-input" autocomplete='off'>
+                                    </span>`
+  }
+  else {
+    spannerCellsList[0].innerHTML = `<span class="column left keyActive keyMargin keyClick" id='keyIcon${tableNumber}${tableColumnNumber}${tableColumnNumber}'>
+                                      <sub>${currSeqId}</sub><img src='./Icons/Icons/ic_vpn_key_24px.svg'>
+                                    </span>
+                                    <span class="column right form-group">
+                                      <input id='columnNameText${tableNumber}${tableColumnNumber}${tableColumnNumber}' type="text" value=${columnNameVal} class="form-control spanner-input" autocomplete='off'>
+                                    </span>`
+  }
+  $('#keyIcon' + tableNumber + tableColumnNumber + tableColumnNumber).click(function () {
+    $(this).toggleClass('keyActive keyNotActive');
+    keyId = $(this).attr('id');
+    for (var z = 0; z < keyColumnMap[tableNumber].length; z++) {
+      if (keyId == keyColumnMap[tableNumber][z].keyIconId) {
+        columnName = keyColumnMap[tableNumber][z].columnName;
+      }
+    }
+
+    if (document.getElementById(keyId).classList.contains('keyActive')) {
+      getNewSeqNumForPrimaryKey(keyId, tableNumber);
+    }
+    else {
+      removePrimaryKeyFromSeq(tableNumber);
+    }
+  });
+}
+
+/**
+ * Function to get new seq number for primary key
+ *
+ * @param {html id} keyId
+ * @param {number} tableColumnNumber
+ * @return {null}
+ */
+const getNewSeqNumForPrimaryKey = (keyId, tableNumber) => {
```

Comment:
tableColumnNumber vs tableNumber 
Line number 853 vs 856

Update Jsdoc to tableNumber

Reply:
comment changes done accordingly